config/crd/bases/tests.testkube.io_testsuites.yaml

```diff
@@ -134,6 +134,101 @@ spec:
                       type: string
                   type: object
                 type: array
+              variables:
+                additionalProperties:
+                  properties:
+                    name:
+                      description: variable name
+                      type: string
+                    type:
+                      description: variable type
+                      type: string
+                    value:
+                      description: variable string value
+                      type: string
+                    valueFrom:
+                      description: or load it from var source
+                      properties:
+                        configMapKeyRef:
+                          description: Selects a key of a ConfigMap.
+                          properties:
+                            key:
+                              description: The key to select.
+                              type: string
+                            name:
+                              description: 'Name of the referent. More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#names
+                                TODO: Add other useful fields. apiVersion, kind, uid?'
+                              type: string
+                            optional:
+                              description: Specify whether the ConfigMap or its key
+                                must be defined
+                              type: boolean
+                          required:
+                          - key
+                          type: object
+                        fieldRef:
+                          description: 'Selects a field of the pod: supports metadata.name,
+                            metadata.namespace, `metadata.labels[''<KEY>'']`, `metadata.annotations[''<KEY>'']`,
+                            spec.nodeName, spec.serviceAccountName, status.hostIP,
+                            status.podIP, status.podIPs.'
+                          properties:
+                            apiVersion:
+                              description: Version of the schema the FieldPath is
+                                written in terms of, defaults to "v1".
+                              type: string
+                            fieldPath:
+                              description: Path of the field to select in the specified
+                                API version.
+                              type: string
+                          required:
+                          - fieldPath
+                          type: object
+                        resourceFieldRef:
```

Comment:
do we use it?

Reply:
No, but it's generated from core data structure (same as used in deployments for env vars)

Reply:
I prefer not to keep them same manually in case of change in the future